messaging/src/main/java/org/axonframework/common/ObjectUtils.java

```diff
@@ -40,7 +43,7 @@ private ObjectUtils() {
      * @param <T>             the type of value to return
      * @return {@code instance} if not {@code null}, otherwise the value provided by {@code defaultProvider}
      */
-    public static <T> T getOrDefault(T instance, Supplier<T> defaultProvider) {
+    public static <T> T getOrDefault(@Nullable T instance, @Nonnull Supplier<T> defaultProvider) {
```

Comment:
This is basically `Objects.requireNonNullElseGet()` no?

Reply:
Sharp! Removed these annotations and used the `Objects` solution at the spot that caused this change.